pkg/common/telemetry/telemetry.go

```diff
@@ -0,0 +1,90 @@
+package telemetry
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/HewlettPackard/Galadriel/pkg/common"
+	runtimemetrics "go.opentelemetry.io/contrib/instrumentation/runtime"
+	"go.opentelemetry.io/otel/exporters/prometheus"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/histogram"
+	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
+	"go.opentelemetry.io/otel/sdk/metric/export/aggregation"
+	processor "go.opentelemetry.io/otel/sdk/metric/processor/basic"
+	selector "go.opentelemetry.io/otel/sdk/metric/selector/simple"
+)
+
+type MetricServer interface {
+	common.RunnablePlugin
+}
+
+type LocalMetricServer struct {
+	logger common.Logger
+}
+
+func NewLocalMetricServer() MetricServer {
+	return &LocalMetricServer{
+		logger: *common.NewLogger(MetricsServer),
+	}
+}
+
+func (c *LocalMetricServer) Run(ctx context.Context) error {
+	c.logger.Info("Starting metric server")
+
+	if err := runtimemetrics.Start(); err != nil {
+		panic(err)
+	}
+
+	// TODO: verify telemetry plugin config to run specified plugin
+	exporter := configurePrometheusMetrics()
+	http.HandleFunc("/metrics", exporter.ServeHTTP)
+
+	go func() {
+		port := "8888"
+		c.logger.Info("Listenening on http://localhost:8888/metrics")
+
+		address := fmt.Sprintf(":%s", port)
+		_ = http.ListenAndServe(address, nil)
```

Comment:
Since we do not care about the error return we could use 
```suggestion
	http.ListenAndServe(address, nil)
```